spring-cloud-aws-s3-parent/spring-cloud-aws-s3/src/main/java/io/awspring/cloud/s3/S3Operations.java

```diff
@@ -47,6 +47,14 @@ public interface S3Operations {
 	 */
 	void deleteBucket(String bucketName);
 
+	/**
+	 * Checks if an S3 bucket exists.
+	 *
+	 * @param bucketName - the bucket name
+	 * @return true if bucket exists; false otherwise
+	 */
+	boolean existsBucket(String bucketName);
```

Comment:
`existsBucket` does not sound right. Perhaps `bucketExists` would be better? @MatejNedic opinion welcome

Reply:
I agree that bucketExists is a much more readable and comprehensive name. Will change it next weekend 🙂

Reply:
Updated the method name to `bucketExists`